vms/shift/views.py

```diff
@@ -1,728 +1,832 @@
-# standard library
```

Comment:
Is there a trailing spaces issue here?

Reply:
Please reslove this so that I can review this file.

Reply:
There is no trailing space issue, don't know what is happening. Is this due to some editor issue, can you help me out.

Reply:
This file change is problem of line endings, accidentally it has been changed. you'll have to use pycharm or terminal cmds to convert back. (you'll have to convert and commit it to see which one it was previously)